### PR TITLE
adding ocr application

### PR DESCRIPTION
**Pull request by eric-capeprivacy**
Adding example application code for OCR, requires
```
RUN apt install -y tesseract-ocr
RUN apt install -y ghostscript
RUN pip install ocrmypdf
```

### CODE REVIEW

**Review by gavinuhma — APPROVED** [review]

File: ocr/app.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
+import os
+import pathlib
+import ocrmypdf
+import sys
+
+
+deploy_dir = os.environ.get("DEPLOY_DIR") or pathlib.Path(__file__).parent
```
Comment (eric-capeprivacy) [inline ocr/app.py:7]: Pretty useless now

File: ocr/app.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+sys.stdout = open('/dev/null', 'w')
+def cape_handler(pdf_bytes: bytes):
+    with open("my_file.pdf", "wb") as binary_file:
+        binary_file.write(pdf_bytes)
+    # doc = doctr.io.DocumentFile.from_pdf(pdf_bytes)
```
Comment (eric-capeprivacy) [inline ocr/app.py:12]: don't need this.

Comment (justin1121) [inline ocr/app.py:12]: delete?

File: ocr/app.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
+import os
+import pathlib
+import ocrmypdf
+import sys
+
+
+deploy_dir = os.environ.get("DEPLOY_DIR") or pathlib.Path(__file__).parent
+sys.stdout = open('/dev/null', 'w')
+def cape_handler(pdf_bytes: bytes):
+    with open("my_file.pdf", "wb") as binary_file:
```
Comment (eric-capeprivacy) [inline ocr/app.py:10]: try using I/O instead.

File: ocr/app.py [hunk 4]
```diff
@@ -0,0 +1,24 @@
+import os
+import pathlib
+import ocrmypdf
+import sys
+
+
+deploy_dir = os.environ.get("DEPLOY_DIR") or pathlib.Path(__file__).parent
+sys.stdout = open('/dev/null', 'w')
```
Comment (justin1121) [inline ocr/app.py:8]: just curious why we're doing this.

**Review by justin1121 — APPROVED** [review]
Couple questions but LGTM

File: ocr/app.py (hunk 4 above)
Comment (eric-capeprivacy) [reply ocr/app.py:8]: This is because writing to the output in the enclave would fail.

**Pull request merged**